front/src/routes/scene/edit-scene/Settings.jsx

```diff
@@ -0,0 +1,93 @@
+import cx from 'classnames';
+import { Localizer, Text } from 'preact-i18n';
+import { useState } from 'preact/compat';
```

Comment:
Except when a lib requires it, we don't use hooks in Gladys frontend for now. 

I'm open to discussing it (I'm not in favor of hooks in general), but it's probably best not to mix code style in the project for now?

Reply:
Fixed